tests/security/apparmor/aa_status.pm

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
```

Comment:
Is it a conscious choice to select the GPL license? BSD as is used in many other files might be a better choice here

Reply:
I copied it from another existed case. And I don't feel GPL license is worse than others.

Reply:
Sure, if you don't mind please use BSD style